### PR TITLE
update jmx2logzio readme

### PR DESCRIPTION
**Pull request by yyyogev**
## Pages to review

- [ ] https://deploy-preview-262--logz-docs.netlify.com/shipping/metrics-sources/jmx.html

## Remaining work

- [ ] Technical review 
- [x] Copy Review @imnotashrimp 

## Post launch

**Do not remove** - To be completed by the docs team upon merge:
- [ ] Teams to update with the new information: Ido, Yogev
- [ ] Copy information to jmx2logzio readme

@imnotashrimp do your magic

### CODE REVIEW

File: _source/logzio_collections/_metrics-sources/jmx.md [hunk 1]
```diff
@@ -157,15 +151,13 @@ consider using [jmx2logzio with Jolokia](#with-a-jolokia-agent-config) instead.
 
 ## jmx2logzio + Jolokia agent setup
 
-If you experience issues such as port conflicts for Docker containers,
-you can run a Jolokia agent to act as a go-between for sending metrics from your app to Logz.io.
-In this configuration, jmx2logzio forwards metrics to Jolokia,
-which sends those metrics to Logz.io.
+If you to poll metrics from a remote source, you can use Jolokia agent + Jmx2logzio
```
Comment (imnotashrimp) [inline _source/logzio_collections/_metrics-sources/jmx.md]:
```suggestion
If you want to poll metrics from a remote source, you can use Jolokia agent + Jmx2logzio.
```

File: _source/logzio_collections/_metrics-sources/jmx.md [hunk 2]
```diff
@@ -157,15 +151,13 @@ consider using [jmx2logzio with Jolokia](#with-a-jolokia-agent-config) instead.
...
+In this configuration, Jolokia exposes the metrics as REST, and jmx2logzio reads it from them and sends it to Logz.io
```
Comment (imnotashrimp) [inline _source/logzio_collections/_metrics-sources/jmx.md]: Not sure what you mean here—can you clarify?

Comment (yyyogev) [reply _source/logzio_collections/_metrics-sources/jmx.md]: The previous description was wrong and misleading. What's actually happens is Jolokia polls the metrics from the target app, expose them as REST API, and once every x seconds jmx2logzio call this api and receive the metrics as answer

Comment (imnotashrimp) [reply _source/logzio_collections/_metrics-sources/jmx.md]:
```suggestion
In this configuration, Jolokia exposes the metrics through an API, which jmx2logzio reads and sends to Logz.io.
```

**Comment by yyyogev** [comment]
Awesome. Now how do we apply it to the repo's readme?

**Comment by imnotashrimp** [comment]
I run my regex and copy it over. :) Doing it now.

**Pull request merged**